src/main/kotlin/apply/ui/api/ExcelController.kt

```diff
@@ -19,9 +19,9 @@ class ExcelController(
     private val recruitmentService: RecruitmentService,
     private val evaluationService: EvaluationService
 ) {
-    @GetMapping("/applicants/excel")
-    fun createApplicantExcel(@PathVariable recruitmentId: Long): ResponseEntity<InputStreamResource> {
-        val excel = excelService.createApplicantExcel(recruitmentId)
+    @GetMapping("/users/excel")
+    fun createUserExcel(@PathVariable recruitmentId: Long): ResponseEntity<InputStreamResource> {
+        val excel = excelService.createUserExcel(recruitmentId)
```

Comment:
`Applicant`가 더 와닿지 않나요?

```suggestion
    @GetMapping("/applicants/excel")
    fun createApplicantExcel(@PathVariable recruitmentId: Long): ResponseEntity<InputStreamResource> {
        val excel = excelService.createApplicantExcel(recruitmentId)
```

Reply:
이 부분도 지원한 유저라는 의미를 담고 있기 때문에 Applicant가 더 어울리다고 생각해 반영했습니다!